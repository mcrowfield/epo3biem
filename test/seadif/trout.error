This is the file 'seadif/trout.error' which was created by
the program 'trout' on Tue Dec 19 11:54:34 2017
Trout was verifying the netlist description of
your circuit circuit 'vga_controller' with the current layout in seadali.
The following nets in the layout are not according to
the netlist description of circuit 'vga_controller'. They are also
marked in the layout.
------------------------- list of errors -------------------------
@@@@@ partially unconnected net: 'h_count_2_port' (1 unconnects among 4)
@@@@@ entirely unconnected net: 'n121' (2 terminals)
@@@@@ partially unconnected net: 'n220' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n183' (1 unconnects among 6)
@@@@@ partially unconnected net: 'v_count_3_port' (1 unconnects among 5)
@@@@@ partially unconnected net: 'v_count_4_port' (2 unconnects among 6)
@@@@@ partially unconnected net: 'h_count_0_port' (1 unconnects among 5)
@@@@@ partially unconnected net: 'n163' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n173' (4 unconnects among 8)
@@@@@ entirely unconnected net: 'h_count_1_port' (11 terminals)
@@@@@ partially unconnected net: 'n178' (3 unconnects among 8)
@@@@@ entirely unconnected net: 'v_count_9_port' (11 terminals)
@@@@@ entirely unconnected net: 'h_count_6_port' (16 terminals)
@@@@@ partially unconnected net: 'n186' (9 unconnects among 16)
@@@@@ partially unconnected net: 'reset' (97 unconnects among 125)
@@@@@ partially unconnected net: 'pixel_clk' (422 unconnects among 435)
@@@@@ entirely unconnected net: 'n156' (92 terminals)

WARNING: 17 out of the 186 nets are not properly connected.
         In the layout each of the unconnects is marked by
         an arrow followed by the name of the net: <=-<net_name>
------------------------------------------------------------------
