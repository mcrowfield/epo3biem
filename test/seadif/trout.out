>>>>>> trout: Automatic Router for Sea-of-Gates           <<<<<<
>>>>>> (c) 1995 Patrick Groeneveld, Delft Univ. of Techn. <<<<<<
------ reading image description file '/data/public/common/software/nelsis/20161101_32/share/lib/celllibs/fishbone/image.seadif'
<<<<<<< HEAD
<<<<<<< HEAD
<<<<<<< HEAD
WARNING: Seadif file "/home/mapostma/Documents/EPO3 Github/epo3biem/test/seadif/sealib2.sdf" is not a magic cookie (skipped).

------ reading placement 'Tmp_Cell_(vga_controller(vga_controller(test8)))'
=======
------ reading placement 'Tmp_Cell_(posnew(posnew(test)))'
>>>>>>> parent of c7e97f1... vga_controller less registers
=======
------ reading placement 'Tmp_Cell_(posnew(posnew(test)))'
>>>>>>> parent of c7e97f1... vga_controller less registers
------ building data structure
------ making grid
------ routing started Tue Dec 19 09:58:23 2017
43 nets have to be routed.
Routing net(1) 'r61_n25' (2/2): . OK!
Routing net(2) 'r61_n4' (2/2): . OK!
Routing net(3) 'r61_n21' (2/2): . OK!
Routing net(4) 'r61_n3' (2/2): . OK!
Routing net(5) 'r61_n15' (2/2): . OK!
Routing net(6) 'r61_n17' (2/2): . OK!
Routing net(7) 'r61_n20' (2/2): . OK!
Routing net(8) 'r61_n22' (2/2): . OK!
Routing net(9) 'r61_n11' (2/2): . OK!
Routing net(10) 'r61_n16' (2/2): . OK!
Routing net(11) 'r61_n23' (3/3): .. OK!
Routing net(12) 'r61_n13' (2/2): . OK!
Routing net(13) 'position_new_5_0_0' (2/2): .b OK!
Routing net(14) 'r61_n19' (2/2): . OK!
Routing net(15) 'position_new_5_0_5' (2/2): .b OK!
Routing net(16) 'position_new_5_0_1' (2/2): .b OK!
Routing net(17) 'position_new_5_0_2' (2/2): .b OK!
Routing net(18) 'position_new_5_0_3' (2/2): .b OK!
Routing net(19) 'position_new_5_0_4' (2/2): .b OK!
Routing net(20) 'position_old_5_0_5' (2/2): .b OK!
Routing net(21) 'position_old_5_0_2' (3/3): ..b OK!
Routing net(22) 'r61_n7' (3/3): .. OK!
Routing net(23) 'r61_n24' (2/2): . OK!
Routing net(24) 'r61_n28' (2/2): . OK!
Routing net(25) 'r61_n14' (2/2): . OK!
Routing net(26) 'r61_n9' (2/2): . OK!
Routing net(27) 'r61_n26' (3/3): .. OK!
Routing net(28) 'r61_n5' (3/3): .. OK!
Routing net(29) 'r61_n12' (4/4): ... OK!
Routing net(30) 'r61_n18' (4/4): ... OK!
Routing net(31) 'r61_n8' (2/2): . OK!
Routing net(32) 'position_old_5_0_3' (4/4): ...b OK!
Routing net(33) 'r61_n6' (3/3): .. OK!
Routing net(34) 'position_old_5_0_0' (3/3): ..b OK!
Routing net(35) 'position_old_5_0_4' (4/4): ...b OK!
Routing net(36) 'r61_n10' (2/2): . OK!
Routing net(37) 'r61_n2' (2/2): . OK!
Routing net(38) 'r61_n27' (2/2): . OK!
Routing net(39) 'n11' (7/7): ...... OK!
Routing net(40) 'button' (7/7): ......b OK!
Routing net(41) 'position_old_5_0_1' (3/3): ..b OK!
Routing net(42) 'n8' (35/35): ..................................b OK!
Routing net(43) 'n9' (35/35): ..................................b OK!
READY
----- Some interesting statistics ------
Total number of nets:                 43 (100% completion)
Total length of routed wires:         1390 grids = 12.232 mm
No. of transistors (total / used):    318/270 = 84.91 %
CPU-time consumption:                 0.00 sec.
Elapsed time during routing:          0.00 sec. (100.00 % of cpu)
------ Verifying Connectivity ------
<<<<<<< HEAD
<<<<<<< HEAD
WARNING: 17 out of the 186 nets are not properly connected.
------ Writing 'Tmp_Cell_(vga_controller(vga_controller(test8)))'
------ Trout: task completed on Tue Dec 19 11:54:34 2017
=======
<<<<<<< HEAD
WARNING: Seadif file "/home/jrkoning/epo3git/epo3biem/test/seadif/sealib2.sdf" is not a magic cookie (skipped).

------ reading placement 'Tmp_Cell_(pos(pos(test2)))'
------ building data structure
------ making grid
------ routing started Tue Dec 19 11:38:07 2017
45 nets have to be routed.
Routing net(1) 'n29' (2/2): . OK!
Routing net(2) 'n32' (2/2): . OK!
Routing net(3) 'n34' (2/2): . OK!
Routing net(4) 'n36' (2/2): . OK!
Routing net(5) 'n38' (2/2): . OK!
Routing net(6) 'next_state_0_port' (2/2): . OK!
Routing net(7) 'n25' (2/2): . OK!
Routing net(8) 'n27' (2/2): . OK!
Routing net(9) 'n41' (2/2): . OK!
Routing net(10) 'n28' (2/2): . OK!
Routing net(11) 'n30' (2/2): . OK!
Routing net(12) 'n33' (2/2): . OK!
Routing net(13) 'n35' (2/2): . OK!
Routing net(14) 'n37' (2/2): . OK!
Routing net(15) 'n39' (2/2): . OK!
Routing net(16) 'clk' (9/9): ........b OK!
Routing net(17) 'n40' (2/2): . OK!
Routing net(18) 'n20' (2/2): . OK!
Routing net(19) 'n21' (2/2): . OK!
Routing net(20) 'n22' (2/2): . OK!
Routing net(21) 'n23' (2/2): . OK!
Routing net(22) 'n24' (2/2): . OK!
Routing net(23) 'n19' (2/2): . OK!
Routing net(24) 'position_new_5_0_0' (2/2): .b OK!
Routing net(25) 'position_new_5_0_1' (2/2): .b OK!
Routing net(26) 'position_new_5_0_2' (2/2): .b OK!
Routing net(27) 'position_new_5_0_3' (2/2): .b OK!
Routing net(28) 'position_new_5_0_4' (2/2): .b OK!
Routing net(29) 'position_new_5_0_5' (2/2): .b OK!
Routing net(30) 'position_old_5_0_1' (3/3): ..b OK!
Routing net(31) 'position_old_5_0_4' (3/3): ..b OK!
Routing net(32) 'position_old_5_0_3' (3/3): ..b OK!
Routing net(33) 'position_old_5_0_2' (3/3): ..b OK!
Routing net(34) 'position_old_5_0_0' (3/3): ..b OK!
Routing net(35) 'enable' (3/3): ..b OK!
Routing net(36) 'state_1_port' (3/3): .. OK!
Routing net(37) 'position_old_5_0_5' (3/3): ..b OK!
Routing net(38) 'n42' (2/2): . OK!
Routing net(39) 'n26' (4/4): ... OK!
Routing net(40) 'n31' (7/7): ...... OK!
Routing net(41) 'count' (3/3): ..b OK!
Routing net(42) 'vdd' :  (Power special)
Routing net(43) 'state_0_port' (10/10): ......... OK!
Routing net(44) 'next_state_1_port' (2/2): . OK!
Routing net(45) 'vss' :  (Power special)
READY
----- Some interesting statistics ------
Total number of nets:                 45 (100% completion)
Total length of routed wires:         2023 grids = 17.802 mm
No. of transistors (total / used):    5220/446 = 8.54 %
CPU-time consumption:                 0.02 sec.
Elapsed time during routing:          0.01 sec. (200.00 % of cpu)
------ Verifying Connectivity ------
------ No unconnect nor short-circuits ------
------ Writing 'Tmp_Cell_(pos(pos(test2)))'
------ Trout: task completed on Tue Dec 19 11:38:07 2017
=======
WARNING: Seadif file "/home/michaelkraaije/epo3git/epo3biem/test/seadif/sealib2.sdf" is not a magic cookie (skipped).

------ reading placement 'Tmp_Cell_(posnew(posnew(test)))'
------ building data structure
------ making grid
------ routing started Tue Dec 19 10:51:52 2017
43 nets have to be routed.
Routing net(1) 'r61_n25' (2/2): . OK!
Routing net(2) 'r61_n4' (2/2): . OK!
Routing net(3) 'r61_n21' (2/2): . OK!
Routing net(4) 'r61_n3' (2/2): . OK!
Routing net(5) 'r61_n15' (2/2): . OK!
Routing net(6) 'r61_n17' (2/2): . OK!
Routing net(7) 'r61_n20' (2/2): . OK!
Routing net(8) 'r61_n22' (2/2): . OK!
Routing net(9) 'r61_n11' (2/2): . OK!
Routing net(10) 'r61_n16' (2/2): . OK!
Routing net(11) 'r61_n23' (3/3): .. OK!
Routing net(12) 'r61_n13' (2/2): . OK!
Routing net(13) 'position_new_5_0_0' (2/2): .b OK!
Routing net(14) 'r61_n19' (2/2): . OK!
Routing net(15) 'position_new_5_0_5' (2/2): .b OK!
Routing net(16) 'position_new_5_0_1' (2/2): .b OK!
Routing net(17) 'position_new_5_0_2' (2/2): .b OK!
Routing net(18) 'position_new_5_0_3' (2/2): .b OK!
Routing net(19) 'position_new_5_0_4' (2/2): .b OK!
Routing net(20) 'position_old_5_0_5' (2/2): .b OK!
Routing net(21) 'position_old_5_0_2' (3/3): ..b OK!
Routing net(22) 'r61_n7' (3/3): .. OK!
Routing net(23) 'r61_n24' (2/2): . OK!
Routing net(24) 'r61_n26' (3/3): .. OK!
Routing net(25) 'r61_n5' (3/3): .. OK!
Routing net(26) 'r61_n12' (4/4): ... OK!
Routing net(27) 'r61_n8' (2/2): . OK!
Routing net(28) 'r61_n18' (4/4): ... OK!
Routing net(29) 'r61_n28' (2/2): . OK!
Routing net(30) 'position_old_5_0_3' (4/4): ...b OK!
Routing net(31) 'position_old_5_0_0' (3/3): ..b OK!
Routing net(32) 'r61_n27' (2/2): . OK!
Routing net(33) 'r61_n14' (2/2): . OK!
Routing net(34) 'r61_n9' (2/2): . OK!
Routing net(35) 'r61_n6' (3/3): .. OK!
Routing net(36) 'r61_n10' (2/2): . OK!
Routing net(37) 'r61_n2' (2/2): . OK!
Routing net(38) 'position_old_5_0_1' (3/3): ..b OK!
Routing net(39) 'button' (7/7): ......b OK!
Routing net(40) 'n11' (7/7): ...... OK!
Routing net(41) 'position_old_5_0_4' (4/4): ...b OK!
Routing net(42) 'n8' (35/35): ..................................b OK!
Routing net(43) 'n9' (35/35): ..................................b OK!
READY
----- Some interesting statistics ------
Total number of nets:                 43 (100% completion)
Total length of routed wires:         1444 grids = 12.707 mm
No. of transistors (total / used):    318/270 = 84.91 %
CPU-time consumption:                 0.00 sec.
Elapsed time during routing:          0.01 sec. (0.00 % of cpu)
------ Verifying Connectivity ------
------ No unconnect nor short-circuits ------
------ Writing 'Tmp_Cell_(posnew(posnew(test)))'
------ Trout: task completed on Tue Dec 19 10:51:52 2017
>>>>>>> 9eaa79fb88a349a7324963bc0eca39907e8f29b0
terminate
>>>>>>> 0e03a5bee655ae6f852fec6836b16c3ee6a3c259
=======
=======
>>>>>>> parent of c7e97f1... vga_controller less registers
------ No unconnect nor short-circuits ------
------ Writing 'Tmp_Cell_(posnew(posnew(test)))'
------ Trout: task completed on Tue Dec 19 09:58:23 2017
terminate
<<<<<<< HEAD
>>>>>>> parent of c7e97f1... vga_controller less registers
=======
>>>>>>> parent of c7e97f1... vga_controller less registers
